Stellar-operation-create-manage-limits-request.x

```diff
@@ -15,56 +15,68 @@ namespace stellar
     Result: CreateManageLimitsRequestResult
 */
 
+//: CreateManageLimitsOp is used to create reviewable request which on approval will update limits set in the system
 struct CreateManageLimitsRequestOp
 {
+    //: Body of the UpdateLimits reviewable request to be created
     LimitsUpdateRequest manageLimitsRequest;
 
-	uint32* allTasks;
+    //: (optional) Bit mask whose flags must be cleared in order for ManageLimits request to be approved, which will be used
```

Comment:
//: (optional) Bit mask whose flags must be cleared in order for ManageLimits request to be approved, which will be used by key `limits_update_tasks`instead of value from the key-value pair

Исправил на вот так. Может можно как то все таки еще понятней сказать

Reply:
Ерунду написал